Remove default, add placeholders (#2)
utopia-php/locale: README.md

use Utopia\Locale\Locale;

// Init translations
Locale::setLanguageFromArray('en-US', [
    'hello' => 'Hello',
    'world' => 'World',
    'likes' => 'You have {likesAmount} likes and {commentsAmount} comments.'
]); // Set English
Locale::setLanguageFromArray('he-IL', ['hello' => 'שלום',]); // Set Hebrew
Locale::setLanguageFromJSON('hi-IN', 'path/to/translations.json'); // Set Hindi

// Create locale instance
$locale = new Locale('en-US'); // en-US will be set as default language

// Get translation
echo $locale->getText('hello'); // prints "Hello"
echo $locale->getText('world'); // prints "World"

// Use placeholders
echo $locale->getText('likes', [ 'likesAmount' => 12, 'commentsAmount' => 55 ]); // prints "You have 12 likes and 55 comments."

> echo $locale->getText('likes'); // prints "You have {likesAmount} likes and {commentsAmount} comments.". If you don't provide placeholder value, we don't touch it

Reviewer: ```suggestion
echo $locale->getText('likes'); // prints "You have {likesAmount} likes and {commentsAmount} comments.". If you don't provide placeholder value, the string is returned unchanged.
```
